### ISSUE TITLE
Error: INSTALLATION FAILED

### ISSUE THREAD

**Issue by XieRW** [kvaps/kubernetes-in-kubernetes#14]
when I use command :
helm install foo kvaps/kubernetes --version 0.13.3   --namespace foo   --create-namespace   --set persistence.storageClassName=local-path

It return this:
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/xrw/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/xrw/.kube/config
W0224 17:14:07.219758  293805 warnings.go:70] batch/v1beta1 CronJob is deprecated in v1.21+, unavailable in v1.25+; use batch/v1 CronJob
W0224 17:14:07.282543  293805 warnings.go:70] batch/v1beta1 CronJob is deprecated in v1.21+, unavailable in v1.25+; use batch/v1 CronJob
Error: INSTALLATION FAILED: Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": dial tcp 10.102.100.71:443: i/o timeout

what shouid i do?

**Comment by kvaps** [comment]
Hi, this was fixed by https://github.com/kvaps/kubernetes-in-kubernetes/pull/11 but didn't get into release.

I just released the new verion, please try `0.13.4`

**Comment by XieRW** [comment]
I change the command as you say ,but it tured:
Error: INSTALLATION FAILED: failed to download "kvaps/kubernetes" at version "0.13.4"

**Comment by XieRW** [comment]
I have solved the problem.

> helm repo update
then:
> helm install foo kvaps/kubernetes --version 0.13.4 --namespace foo --create-namespace --set persistence.storageClassName=local-path

thanks.

**Comment by XieRW** [comment]
> 
another question:

Error: INSTALLATION FAILED: Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": context deadline exceeded

**Comment by XieRW** [comment]
> Hi, this was fixed by #11 but didn't get into release.
> 
> I just released the new verion, please try `0.13.4`

another question:

Error: INSTALLATION FAILED: Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "[https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s](https://cert-manager-webhook.cert-manager.svc/mutate?timeout=10s)": context deadline exceeded

**Comment by kvaps** [comment]
Something wrong with your cert-manager. Try inspecting pods in cert-manager namespace:

```
kubectl get pod -n cert-manager
```

and webhooks itself:

```
kubectl get mutatingwebhookconfigurations
kubectl get validatingwebhookconfigurations
```
